### PR TITLE
prepare v2.24.0

### PR DESCRIPTION
**Pull request by Issif**
Signed-off-by: Issif <issif_github@gadz.org>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area build

> /area config

/area outputs

> /area tests

**What this PR does / why we need it**:

This PR is for release v2.24.0

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Issif** [comment]
/do-not-merge/hold

File: CHANGELOG.md [hunk 1]
```diff
@@ -297,7 +306,7 @@ This release is the last one with an Helm chart, the next ones will be in [Falco
 
 ## 1.0.3 - 2019-01-30
 #### New
-- New output  : **Alert Manager**
+- New output : **Alert Manager**
```
Comment (developer-guy) [inline CHANGELOG.md:309]: whitespace here :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,23 @@
 # Changelog
 
+## 2.24.0 - 2021-08-13
+#### New
+- New output: **Grafana** ([PR#254](https://github.com/falcosecurity/falcosidekick/pull/254))
+- New output: **Fission** ([PR#255](https://github.com/falcosecurity/falcosidekick/pull/254) thanks to [@gauravgahlot](https://github.com/gauravgahlot))
```
Comment (developer-guy) [inline CHANGELOG.md:6]: I think there is a wrong reference here, it should be https://github.com/falcosecurity/falcosidekick/pull/255.

**Review by developer-guy — CHANGES_REQUESTED** [review]

**Review by developer-guy — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
 
+## 2.24.0 - 2021-08-13
+#### New
+- New output: **Grafana** ([PR#254](https://github.com/falcosecurity/falcosidekick/pull/254))
+- New output: **Fission** ([PR#255](https://github.com/falcosecurity/falcosidekick/pull/255) thanks to [@gauravgahlot](https://github.com/gauravgahlot))
+- New output: **Yandex Cloud S3** ([PR#261](https://github.com/falcosecurity/falcosidekick/pull/261) thanks to [@nar3k](https://github.com/nar3k))
+- New output: **Kafka REST** ([PR#263](https://github.com/falcosecurity/falcosidekick/pull/263) thanks to [@dirien](https://github.com/dirien))
+#### Enhancement
+- Set header `x-amz-acl` to `bucket-owner-full-control` for output `AWS S3` ([PR#264](https://github.com/falcosecurity/falcosidekick/pull/264) thanks to [@Kaizhe](https://github.com/Kaizhe))
+- Docker image is now availagle on [`AWS ECR Public Gallery`](https://gallery.ecr.aws/falcosecurity/falcosidekick) ([PR#265](https://github.com/falcosecurity/falcosidekick/pull/265) thanks to [@maxgio92](https://github.com/maxgio92))
```
Comment (developer-guy) [inline CHANGELOG.md:11]: `availagle`, typo in this word.

**Review by developer-guy — CHANGES_REQUESTED** [review]

**Review by developer-guy — APPROVED** [review]
LGTM

**Pull request merged**